adhocracy4/actions/migrations/0006_set_action_target_creator.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.19 on 2023-05-25 08:22
+
+from django.db import migrations
+
+
+def set_action_target_creator(apps, schema_editor):
+    Action = apps.get_model("a4actions", "Action")
+    for action in Action.objects.filter(verb="add"):
+        if hasattr(action, "target_content_type") and action.target_content_type:
+            model = apps.get_model(
+                app_label=action.target_content_type.app_label,
+                model_name=action.target_content_type.model,
+            )
+            content_object = model.objects.get(id=action.target_object_id)
```

Comment:
Couldnt you just get it with action.target?

Reply:
no, generic relations / fks are not available in migrations :/

Reply:
Ah, right! And annoyoing!